src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerCFTTest.java

```diff
@@ -304,7 +304,7 @@ public void should_return_a_200_with_task_and_correct_properties() {
             .body("task.case_name", notNullValue())
             .body("task.auto_assigned", notNullValue())
             .body("task.warnings", notNullValue())
-            .body("task.permissions.values", hasItems("Read", "Refer", "Own", "Manage", "Cancel"))
+            .body("task.permissions.values", hasItems("Read", "Refer", "Own"))
             .body("task.description", notNullValue())
```

Comment:
as per requirement, we will need to change the matcher to check this contains only the permissions relevant to the user